server/models.py

```diff
@@ -47,3 +50,41 @@ def __init__(self, *data, **kwargs):
 
     def __repr__(self):
         return "<USER: {}".format(self.username)
+
+class Client(dbs.Model):
+    id = dbs.Column(dbs.Integer, primary_key=True)
+    name = dbs.Column(dbs.String)
+    dob = dbs.Column(dbs.Date)
+    # 123-456-7890
+    phone = dbs.Column(dbs.VARCHAR(12))
+    address = dbs.Column(dbs.String)
+    city = dbs.Column(dbs.String)
+    state = dbs.Column(dbs.String)
+    # 64111 or 64111-0000
+    zip_code = dbs.Column(dbs.VARCHAR(10))
+    status = dbs.Column(dbs.String)
```

Comment:
Is `Client.status` a free string or set of status types?

Reply:
For now, a string works.  We do not know what the status will be and will save us the time of setting up a foreign key and table.